### PR TITLE
Contributing PROJ Cartographic Projections Library

### PR DESCRIPTION
**Pull request by tloizou**

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
@tloizou: missing `easyblock = 'ConfigureMake'` in PROJ easyconfigs

File: easybuild/easyconfigs/p/PROJ/PROJ-4.8.0-goolf-1.4.10.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    geographic longitude and latitude coordinates into cartesian coordinates"""
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+toolchainopts = {'opt': True, 'pic': True, 'usempi': True}
+
+source_urls = ['http://download.osgeo.org/proj/']
+sources = [SOURCELOWER_TAR_GZ]
+
+sanity_check_paths = {
+    'files': ['bin/cs2cs','bin/geod','bin/invgeod','bin/invproj','bin/nad2bin','bin/proj'],
```
Comment (ocaisa) [inline easybuild/easyconfigs/p/PROJ/PROJ-4.8.0-goolf-1.4.10.eb]: Missing whitespace after commas

File: easybuild/easyconfigs/p/PROJ/PROJ-4.8.0-ictce-5.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    geographic longitude and latitude coordinates into cartesian coordinates"""  
+
+toolchain = {'name': 'ictce', 'version': '5.3.0'}
+toolchainopts = {'opt': True, 'pic': True, 'usempi': True}
+
+source_urls = ['http://download.osgeo.org/proj/']
+sources = [SOURCELOWER_TAR_GZ]
+
+sanity_check_paths = {
+    'files': ['bin/cs2cs','bin/geod','bin/invgeod','bin/invproj','bin/nad2bin','bin/proj'],
```
Comment (ocaisa) [inline easybuild/easyconfigs/p/PROJ/PROJ-4.8.0-ictce-5.3.0.eb]: Missing whitespace after commas

**Comment by tloizou** [comment]
@boegel: Ready for testing

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/e9912f2ce167e3b1d24f for a full test report.

**Comment by boegel** [comment]
Looking good, thanks @tloizou!

**Pull request merged**